### PR TITLE
Rough Draft - CPS File Documentation

### PR DESCRIPTION
**Pull request by andersonfrailey**
I've put together a draft of the documentation to go along with the CPS file. It's still a work in progress so please forgive any typos, but I'd appreciate any feedback y'all may have on what I've written so far. I'm particularly looking for sections need clarification or information I may have neglected to add.

The section on Tax-Calculator integration is intentionally left blank because we haven't totally established exactly how that will go yet.

@Amy-Xu @MattHJensen @martinholmer @hdoupe

### CODE REVIEW

**Comment by martinholmer** [comment]
@andersonfrailey, I think your draft CPS data documentation in #109 looks good overall.  I have only a few (relatively minor) suggestions, mostly at the beginning.  Here they are:

(1) @MattHJensen just dropped OSPC from the Tax-Calculator documentation in [Tax-Calculator 1481](https://github.com/open-source-economics/Tax-Calculator/pull/1481), so I'm guessing this:
> such as the Open Source Policy Center's (OSPC) Tax-Calculator,

should be simply:
> such as Tax-Calculator,

(2) The first paragraph in the Overview section now says this:
> The March supplement to the Current Population Survey (CPS) is an annual survey of US households conducted in March of every year. The file contains information on income, demographic, and family structure that can be used to create tax units comprised of those surveyed. Our final file is created using the 2013, 2014, and 2015 CPS supplements. The 2013 and 2014 files are then aged to 2015, making the file representative of tax year 2014.

How about something like this:
> The Annual Social and Economic Supplement (ASEC) to the Current Population Survey (CPS) is an annual survey of US households conducted in March of every year. The survey contains information on prior-year income and current family structure that can be used to create tax units comprised of those surveyed. Our final file is created using the 2013, 2014, and 2015 March CPS supplements. Dollar amounts in the 2013 and 2014 files are then aged to be compatible with values in the 2015 file, making the combined file representative of tax year 2014.

(3) Now the draft says this:
> Our second uses the Open Source CPS Transfer Augmentation Model (C-TAM) to add imputed benefit participation and amounts for Supplemental Security Income (SSI), Supplemental Nutrition Assistance Program (SNAP), Medicare, Medicaid, Social Security, and Veterans' Benefits. 

Given Matt's desire for an `other_ben` variable, wouldn't this be more accurate:
> Our second modification uses the open-source CPS Transfer Augmentation Model (C-TAM) to add imputed benefit participation and imputed benefit amounts for Supplemental Security Income (SSI), Supplemental Nutrition Assistance Program (SNAP), Medicare, Medicaid, Social Security, Veterans' Benefits, and the total of several other benefits.

That's it.  Good work.

**Comment by andersonfrailey** [comment]
Thanks for the comments @martinholmer. I've made the changes in the latest commit.

File: cps_data/cps_file_doc.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+# Current Population Survey Tax Unit File Documentation
+
+Microsimulation models, such as
+[Tax-Calculator](https://github.com/open-source-economics/Tax-Calculator),
+rely on datasets comprised of individual tax units. These datasets are available
+through the IRS, but at substantial financial cost and are not permitted to be
+shared. This document outlines the creation of a tax unit dataset created from
+publicly available CPS files that is suitable for use with OSPC's Tax-Calculator.
```
Comment (MattHJensen) [inline cps_data/cps_file_doc.md]: Drop "OSPC's"

File: cps_data/cps_file_doc.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+[made available by the IRS](https://www.irs.gov/uac/soi-tax-stats-individual-statistical-tables-by-size-of-adjusted-gross-income),
+we determine the distribution of each targeted variable as reported for the
+year 2014. The CPS records are then grouped into 19 bins based on total
+income.<sub>2</sub> Using the distribution found in the IRS data, we determine
+the ideal weighted sum in each income bin. This goal amount is then compared to
+the actual amount in each bin and a ratio is established for each bin. The
+records in each bin are then multiplied by their respective ratios so that the
+distribution more closely matches what is seen in the IRS data.
+
+### Tax-Calculator Integration
```
Comment (MattHJensen) [inline cps_data/cps_file_doc.md]: Let's drop the tax-calculator integration section from this doc, which is about the file's prep.

**Comment by MattHJensen** [comment]
@andersonfrailey, this is excellent. I left a couple of comments. Would love for you to drop the WIP tag when you're ready so we can merge this. 

Over the next weeks/months, let's move towards referencing code in TaxData rather than referencing John's work. Ideally we get to a point where John's work receives a single reference and we describe the TaxData programs that prepare this file as a descendent of the Quantria program.

**Comment by andersonfrailey** [comment]
Latest commit works in the changes suggested by @MattHJensen. I've removed the "WIP" tag from this PR and will be merging it at the end of the day if there are no more comments.

**Pull request merged**